src/pages/layer-2.js

```diff
@@ -0,0 +1,1170 @@
+        description={"Introduction page to layer 2"}
+      />
+
+      <HeroBackground>
+        <HeroContainer>
+          <Hero content={heroContent} isReverse />
+        </HeroContainer>
+
+        <PaddedContent>
+          <StatsContainer>
+            <StatBox>
+              {/* TODO: remove hardcoded StatPrimary once lambda functions are working*/}
+              <StatPrimary>$7.27B</StatPrimary>
+              {/* <StatPrimary>{tvl}</StatPrimary>  */}
+              <StatDescription>TVL locked in layer 2 (USD)</StatDescription>
+            </StatBox>
+            <StatDivider />
+            <StatBox>
+              <StatPrimary>{averageFee}</StatPrimary>
+              <StatDescription>
+                Average layer 2 ETH transfer fee (USD)
+              </StatDescription>
+            </StatBox>
+            <StatDivider />
+            <StatBox>
+              {/* TODO: remove hardcoded StatPrimary once lambda functions are working*/}
+              <StatPrimary>33.01%</StatPrimary>
+              {/* <StatPrimary>{percentChangeL2}</StatPrimary> */}
+              <StatDescription>Layer 2 TVL change (30 days)</StatDescription>
+            </StatBox>
+          </StatsContainer>
+        </PaddedContent>
+      </HeroBackground>
+
+      <PaddedContent id="what-is-layer-2">
+        <TwoColumnContent>
+          <Flex50>
+            <h2>What is layer 2?</h2>
+            <p>
+              Layer 2 (L2) is a collective term to describe a specific set of
+              Ethereum scaling solutions.{" "}
+              <b>
+                A layer 2 is separate blockchain that extends Ethereum and
+                inherits the security guarantees of Ethereum
+              </b>
+              .
+            </p>
+            <p>
+              Now let’s dig into it a bit more, and to do this we need to
+              explain layer 1 (L1).
+            </p>
+          </Flex50>
+          <Flex50>
+            <GatsbyImage
+              image={getImage(data.whatIsEthereum)}
+              style={{ maxHeight: "400px" }}
+              objectFit="contain"
+            />
+          </Flex50>
+        </TwoColumnContent>
+      </PaddedContent>
+      <LightGrayContent>
+        <h2>Layer 1 function</h2>
+        <TwoColumnContent>
+          <Flex50>
+            <p>
+              Layer 1 is the base blockchain. Ethereum and Bitcoin are both
+              layer 1 blockchains because they are the{" "}
+              <b>
+                underlying foundation that various layer 2 networks build on top
+                of
+              </b>
+              . Examples of layer 2 projects include "rollups" on Ethereum and
+              the Lighting Network on top of Bitcoin. All user transaction
+              activity on these layer 2 projects can ultimately settle back to
+              the layer 1 blockchain.
+            </p>
+            <p>
+              Ethereum also functions as a data availability layer for Layer 2s,
```

Comment:
```suggestion
              Ethereum also functions as a data availability layer for layer 2s,
```
`layer 2` not `Layer 2`, right? We should double-check we're never referencing it as a proper noun.